data/tutorials/gs_00_up_and_running.md

```diff
@@ -11,88 +11,165 @@ date: 2021-05-27T21:07:30-00:00
+
+Although this is theoretically all we need to write OCaml code, it is not a complete and comfortable development environment.
+
+## Setting Up Development Tools
 
-### For Windows
+We will now install everything we need to get a complete development environment, which includes:
 
-OCaml on Windows is still a work in progress. In the meantime, if you only
-need to _run_ OCaml programs on a Windows machine, then the simplest solution is to use the Windows Subsystem for Linux 2 (WSL2). WSL2 is a feature that allows Linux programs to run directly on Windows. WSL2 is substantially easier and faster to use than WSL1. Microsoft has comprehensive installation steps for [setting up WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+- Dune, a fast and full-featured build system for OCaml
+- Merlin (the backend) and `ocaml-lsp-server` (OCaml's Language Server Protocol) to provide editors with many useful features such as "jump to definition"
+- `odoc` to generate documentation from OCaml code
+- OCamlformat to automatically format OCaml code
+- UTop, an improved REPL
+- `dune-release` to release code to `opam-repository`, the package base for opam.
 
-On the other hand, if you need Windows binaries, you will have to use the [OCaml for Windows](https://fdopen.github.io/opam-repository-mingw/) installer which comes in 32bit and 64bit versions. This installer gives you Opam and OCaml installations all in one go. It's used from within a Cygwin environment, but the executables produced have no dependency on Cygwin at all. For a more comprehensive update on the state of OCaml on Windows, see the [OCaml on Windows](/docs/ocaml-on-windows) page on the old version of the site.
+All these tools can be installed in your current switch (remember that opam groups installed packages in independent switches) using the following command:
 
-## The OCaml Toplevel
+```shell
+$ opam install dune merlin ocaml-lsp-server odoc ocamlformat utop dune-release
+```
+
+Now that the tools are installed, it remains to understand how to use them. Most of them will be driven either by the editor or by Dune, but UTop is handy to try OCaml or a specific library.
 
-_Toplevel_ is a read-eval-print loop (REPL). It's one of the things that makes OCaml so efficient because it compiles while you code, allowing for iteration.
+## Using the OCaml Toplevel with UTop
 
-OCaml comes with two additional compilers: one compiles to **native code** (sometimes called machine code or executable binary), directly read by the CPU, and the other compiles to **bytecode**, creating an executable that can be interpreted by a variety of runtime environments, making more flexible.
+UTop is a nice toplevel for OCaml. Unlike the standard toplevel with the `ocaml` command, UTop features history, tab completion, line edition, and the ability to load a package installed in your switch. One of the things that makes OCaml so efficient is because it compiles while you code, allowing for iteration.
```

Comment:
Nitpick: I'm not sure what the last sentence in this paragraph means, and I think the grammar is slightly off.